estuary_updater/handlers/errata.py

```diff
@@ -201,7 +156,7 @@ def builds_added_handler(self, msg):
             'id_': msg['body']['headers']['errata_id']
         })[0]
 
-        koji_build = self.get_or_create_build(msg)
+        koji_build = self.get_or_create_build(msg['body']['headers']['brew_build'])
```

Comment:
The 'brew_build' name could be a bit confusing - maybe specify somewhere that this is an nvr?